todolist/app/screens/BookDetail/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import Detail from './layout';
+
+class BooKDetail extends Component {
+  static navigationOptions = ({ navigation }) => {
```

Comment:
what's this for?

Reply:
get the title of the book and put it in the header